ui/src/layouts/chats/mod.rs

```diff
@@ -95,6 +98,46 @@ pub fn ChatLayout(cx: Scope) -> Element {
         div {
             id: "chat-layout",
             aria_label: "chat-layout",
+            tabindex: "0",
+            onkeydown: move |e: Event<KeyboardData>| {
+                // HACK(Linux): Allow copy and paste files for Linux
+                if cfg!(target_os = "linux") {
+                    let keyboard_data = e;
+                    if keyboard_data.code() == Code::KeyV
+                        && keyboard_data.modifiers() == Modifiers::CONTROL
+                    {
+                        cx.spawn({
+                            to_owned![state];
+                            async move {
+                                let files_local_path = tokio::task::spawn_blocking(|| {
+                                    get_files_path_from_clipboard().unwrap_or_default()
+                                })
+                                .await
+                                .expect("Should succeed");
+                                if !files_local_path.is_empty() {
+                                    let new_files: Vec<Location> = files_local_path
+                                    .iter()
+                                    .map(|path| Location::Disk { path: path.clone() })
+                                    .collect();
+                                let mut current_files: Vec<_> = state
+                                    .read()
+                                    .get_active_chat()
+                                    .map(|f| f.files_attached_to_send)
+                                    .unwrap_or_default()
+                                    .drain(..)
+                                    .filter(|x| !new_files.contains(x))
+                                    .collect();
```

Comment:
although this seems fine to me, it would reorder the attachments if you copy/paste files which are already attached, by moving the files to the end of the list. It may be even better to filter `files_local_path` (not the best name btw - `pasted_files` or `to_attach` would be better) by `files_attached_to_send` instead.